packages/auth0-server-js/src/server-client.ts

```diff
@@ -295,6 +296,28 @@ export class ServerClient<TStoreOptions = unknown> {
     };
   }
 
+  /**
+   * Performs a Client-Initiated Backchannel Authentication and returns the tokens.
```

Comment:
Can we add a caveat message here that this will not store the token? As this can be confusing as all other getToken methods do store the tokens.